raw_map/src/lib.rs

```diff
@@ -353,6 +353,11 @@ pub struct RawRoad {
     /// Is there a tagged crosswalk near each end of the road?
     pub crosswalk_forward: bool,
     pub crosswalk_backward: bool,
+    /// Barrier nodes along this road's original center line.
```

Comment:
I'm not sure what would be ideal, but it raises a couple of thoughts. The "short cycle-way" representation should result in the correct behaviour, just based on normal processing of that network. But your LTN editor wants to treat the whole configuration as a single logical "modal filter" for editing and displaying. I think that kind of duality is inevitable (with some algorithms wanting the details, some wanting the logical groupings) and should be part of our design -- like RoadWay vs Corridor and Intersection vs Junction.

So: how would a modal filter work as a point? Representing it as an attribute of a graph edge is a reasonable thing to do: when routing or traversing the graph, you need to calculate an "edge weight" determining the cost of taking that edge. Calculating an infinite cost because there is a modal filter in the middle seems sensible to me. And routing needs to make sense of different positions along an edge (for instance, most real driving trips would start/end part way along a street, not at an intersection).

Another way to represent a modal filter as a point is as an intersection with "turn restrictions" that forbid a car on one side of the filter from reaching the other side. It seems like turn restrictions could already be configured to represent that situation.

I've been thinking about related shortcomings of the current `streets` representation and wrinkles in the whole graph conception that I'll add to a-b-street/osm2streets#1